source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -291,8 +348,32 @@ content: |
              status: 0,
              masterKey: { provider: 'local' }
            }
+     .. tab::
+        :tabid: python
 
+        .. code-block:: python
+           :emphasize-lines: 4
 
+           key = client.encryption.keystore.find_one({ "_id": data_key})
+           client_encryption.close()
+           client.close()
```

Comment:
Could this snippet explicitly print out the value of `key` similar to the other driver examples?

Reply:
Fixed:
- printed the key